### PR TITLE
CVE-2017-14062: bump bitname/kubectl 1.16 images

### PR DESCRIPTION
**Pull request by hectorj2f**
**What type of PR is this?**

Fix
**What this PR does/ why we need it**:

In related to CVE-2017-14062, used in bitnami/kubectl@1.16 is normally used in one-liners to talk to the API. I am not sure how critical is the usage of libidn2 for those one-run calls. 

```
Integer overflow in the decode_digit function in puny_decode.c in Libidn2 before 2.0.4 allows remote attackers to cause a denial of service or possibly have unspecified other impact.
```

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: stable/kommander/charts/kommander-karma/templates/cleanup-configmap.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
       serviceAccountName: {{ template "kommander-karma.sa-name" . }}
       containers:
         - name: kubectl
-          image: bitnami/kubectl:1.16.2
+          image: bitnami/kubectl:1.19.2
```
Comment (jimmidyson) [inline stable/kommander/charts/kommander-karma/templates/cleanup-configmap.yaml:20]: Perhaps use v1.19.7 across the board here for latest 1.19 version?

**Comment by mhrabovcin** [comment]
I can confirm that CVE is not reported in the bumped image:

```
14:47:18 martin@workstation ~$ anchore image vuln bitnami/kubectl:1.19.2 all |  grep CVE-2017-14062
14:47:18 martin@workstation ~$
```

**Review by jimmidyson — APPROVED** [review]

**Review by armandgrillet — APPROVED** [review]

File: stable/kommander/templates/grafana/opsportal-username-secret.yaml [hunk 2]
```diff
@@ -19,7 +19,7 @@ spec:
       containers:
         - name: kubectl
           # --export flag is deprecated so we need to stick with this kubectl version
-          image: bitnami/kubectl:1.16.2
+          image: bitnami/kubectl:1.19.7
```
Comment (gracedo) [inline stable/kommander/templates/grafana/opsportal-username-secret.yaml:22]: let me check into this one, we are using the `--export` flag which is deprecated in newer kubectl versions

File: stable/kommander/templates/grafana/opsportal-username-secret.yaml [hunk 3]
```diff
@@ -19,7 +19,7 @@ spec:
...
           imagePullPolicy: IfNotPresent
           command:
             - /bin/sh
```
Comment (gracedo) [inline stable/kommander/templates/grafana/opsportal-username-secret.yaml:25]: actually we don't need to use to `--export` here (line 27), I think maybe it was left over from a precious implementation that was copying the entire secret over.

Comment (gracedo) [reply stable/kommander/templates/grafana/opsportal-username-secret.yaml:24]: pushed the fix to remove https://github.com/mesosphere/charts/pull/962/commits/288d9106540ff3a5b4443f28dbcef9682236ff5c

**Review by gracedo — APPROVED** [review]

**Pull request merged**